ts/sagas/profile.ts

```diff
@@ -97,35 +97,40 @@ function* createOrUpdateProfileSaga(
         accepted_tos_version: tosVersion,
         version: 0
       };
-
-  const response: SagaCallReturnType<typeof createOrUpdateProfile> = yield call(
-    createOrUpdateProfile,
-    {
+  try {
+    const response: SagaCallReturnType<
```

Comment:
I don't know what is happening with the linter, since no changed have been made about this, but it signs a few errors like this
![Schermata 2019-09-10 alle 18 21 18](https://user-images.githubusercontent.com/822471/64631656-d676bf00-d3f7-11e9-9634-ba3830c8bc33.png)
After some investigation the problem could be the return type of these function: I tried to change from **Iterable<...>**  to **_IterableIterator<...>_** and the errors go away

Do you have these errors too in your editor?

If yes, could you investigate if the above proposed solution could work?

Reply:
Hi @Undermaken , I made some test and from my editor but this issue is not present. Also from @Edigio  editor's doesn't show any lint error.